docs/source/ops/requirements.rst

```diff
@@ -0,0 +1,28 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ops-ref_index:
+
+System Requirements
+===================
+
+Unless otherwise stated, all Daml runtime components require the following dependencies:
+
+1. An x86-compatible system running a modern Unix, Windows, or MacOS operating system.
+2. Java 11 or greater.
+3. An RDBMS system,
+  
+  1. Either PostgreSQL 10.0 or greater.
+  2. Or Oracle Database 19.11 or greater.
+
+Daml is tested using the following specific dependencies in default installations.
+
+1. Opersting Systems:
+  
+  1. Ubuntu 20.04
+  2. Windows Server 2016
+  3. MacOS 10.15 Catalina
+
+2. `Eclipse Adoptium <https://adoptium.net>`_ version 11 for Java.
```

Comment:
not quite what we test with on CI (we get it from nix) but it’s an openjdk 11 so probably close enough

Reply:
Adoptium is the name of the project, but their JVMs are named "Eclipse Temurin".